tests/ImageSharp.Tests/TestImages.cs

```diff
@@ -134,8 +134,9 @@ public static class Bmp
             public const string NegHeight = "Bmp/neg_height.bmp";
             public const string CoreHeader = "Bmp/BitmapCoreHeaderQR.bmp";
             public const string V5Header = "Bmp/BITMAPV5HEADER.bmp";
-            
-            public static readonly string[] All = { Car, F, NegHeight, CoreHeader, V5Header };
+            public const string RLE = "Bmp/RunLengthEncoded.bmp";
+
+            public static readonly string[] All = { Car, F, NegHeight, CoreHeader, V5Header, RLE };
```

Comment:
This has nothing to do with this PR, but I leave my notes here as a reminder:
Adding an image to `TestImages.Bmp.All` adds a new test case to `BmpTests` (so 👍 for adding test coverage for your work @georgexcollins), but we should make `TestImages.Bmp.All` local to the `BmpTests` class, because the current situation is hard to follow.